client-app/src/pages/Login.jsx

```diff
@@ -0,0 +1,98 @@
+import Avatar from "@mui/material/Avatar";
+import Button from "@mui/material/Button";
+import CssBaseline from "@mui/material/CssBaseline";
+import TextField from "@mui/material/TextField";
+import FormControlLabel from "@mui/material/FormControlLabel";
+import Checkbox from "@mui/material/Checkbox";
+import Link from "@mui/material/Link";
+import Grid from "@mui/material/Grid";
+import Box from "@mui/material/Box";
+import LockOutlinedIcon from "@mui/icons-material/LockOutlined";
+import Typography from "@mui/material/Typography";
+import Container from "@mui/material/Container";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+import api from "../api/auth";
+
+const defaultTheme = createTheme();
```

Comment:
[Using with Theme UI](https://mui.com/material-ui/guides/theme-scoping/#using-with-theme-ui)
I think it should be in the root of the app unless you want to change the component specific theme.